Mohan/vector.cpp

```diff
@@ -0,0 +1,38 @@
+// C++ program to illustrate the 
```

Comment:
if you lift code from https://www.geeksforgeeks.org/vector-in-cpp-stl/  here at least care to remove the comments so that you won't get caught  
@akshayrajp please check

Reply:
@eyeamkd what do you mean by caught? Is this a competition? It is for practice I understand the code and executed it so, I pushed same thing.

Reply:
Why would you need a repository if you copy paste the same code from the website?

Reply:
No, I am busy in some other works. I know it's not correct and I didn't mean to copy paste my situation made me do that 🤦🏻‍♂️. So, I did it. I will find time and implement in my own and I will push.

Reply:
That's Okay, close your Pull Request and place one when you are free to implement an STL vector

Reply:
Yeah! Sure